src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -119,14 +119,58 @@
  * &lt;module name="NPathComplexity"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // OK, NPath complexity is less than default threshold
+ *   // function consists of one if-else block with an NPath Complexity of 3
+ *   if (a &gt; 10) {
+ *     if (a &gt; b) { // nested if-else decision tree adds 2 to the complexity count
+ *       baz();
+ *     } else {
+ *       bar();
+ *     }
+ *   } else { // last possible outcome of the main if-else block, adds 1 to complexity
+ *     buzz();
+ *   }
+ * }
+ * </pre>
+ * <p>
  * To configure the check with a threshold of 1000:
  * </p>
  * <pre>
  * &lt;module name="NPathComplexity"&gt;
  *   &lt;property name="max" value="1000"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public void foo() { // violation, NPath complexity is 1,024 (max allowed is 1,000)
+ *   // choosing from 2 options, 10 separate times produces 2^10 (1,024) possible outcomes
```

Comment:
Please update `2 options, 10 separate times` to `2 options 10 times`

Reply:
done